events/2016/paris.html

```diff
@@ -43,7 +43,7 @@ <h1 class="logo center-text main-logo">NodeTogether</h1>
         <h1 class="subtitle center-text">Paris, France</h1>
           <h3 class="center-text">Apply Now!</h3>
           <div class="calls-to-action center-text">
-            <a href="http://goo.gl/forms/bzUiKynzGK" class="btn">Student</a>   
```

Comment:
Removing the spaces from all these places (that's a fun rhyme) would also be another example of a good thing to seperate into yet another PR. Setting each concern into its own PR dramatically increases speed of decision for each PR.

For example, if we actually _didn't_ want to change these spaces (and that was really important to us), we'd have to turn down the PR even though you had really awesome content added (Boston page - woo!).